engine/src/main/java/org/hibernate/validator/HibernateValidatorConfiguration.java

```diff
@@ -87,6 +92,15 @@
 	String SCRIPT_EVALUATOR_FACTORY_CLASSNAME = "hibernate.validator.script_evaluator_factory";
 
 	/**
+	 * Property for configuring clock skew tolerance, allowing to set a margin of error in milliseconds, which is allowed
+	 * when comparing date/time in time related constraints.
```

Comment:
Property for configuring clock skew tolerance, allowing to set the acceptable margin of error when comparing date/time in time related constraints. In milliseconds.